cpp/log/leveldb_db-inl.h

```diff
@@ -47,18 +47,37 @@ std::unique_ptr<const leveldb::FilterPolicy> BuildFilterPolicy() {
 #endif
 
 
-std::string FormatSequenceNumber(const int64_t seq) {
-  return std::to_string(seq);
+std::string IndexToKey(int64_t index) {
+  CHECK_EQ(8, sizeof(index));
```

Comment:
If they're nuts enough to do that then they'll likely just change the `CHECK` too - I'd suggest replacing the `CHECK` with a comment that changing this type will break your stuff.